src/lambda/user.ts

```
@@ -0,0 +1,97 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { DynamoDBService } from "/opt/db/nodejs/service";
+import { UserEntity } from "/opt/types/nodejs/entities";
+import { CurrencyCode } from "/opt/types/nodejs/currency";
+
+import { v4 as uuidv4 } from "uuid";
+
+const DEFAULT_ENTITIES_TABLE_NAME = "dev_playground-service_entities";
+
+export const getUserHandler: APIGatewayProxyHandler = async (event) => {
+  const entitiesDB = new DynamoDBService<UserEntity>(
+    process.env.ENTITIES_TABLE_NAME || DEFAULT_ENTITIES_TABLE_NAME
```

Rather than falling back to a default, I think we should throw an error if `process.env.ENTITIES_TABLE_NAME` is `undefined`. We could have a nice helper function like this:

```typescript
export const getEnvironmentVariable = (name: string) => {
  const value = process.env[name];
  if (value === undefined) throw new Error(`The environment variable ${name} is not defined`);
  return value;
};
```